bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -1514,15 +1514,15 @@ Map_Bookmark_Button_Rename                 = &Rename
 Map_Bookmark_Column_Bearing2               = Dir \u00B0
 Map_Bookmark_Column_Bearing2_Tooltip       = Direction
 Map_Bookmark_Column_Latitude               = Lat
-Map_Bookmark_Column_Latitude_Tooltip       = Mapcenter Latitude
+Map_Bookmark_Column_Latitude_Tooltip       = Latitude
 Map_Bookmark_Column_Longitude              = Lon
-Map_Bookmark_Column_Longitude_Tooltip      = Mapcenter Longitude
+Map_Bookmark_Column_Longitude_Tooltip      = Longitude
```

Comment:
Why was `Mapcenter` removed ?

Reply:
Because when i analyzed it, the word center is neither in the variable name nor in the text of other languages so i assumed it was a mistake?

Reply:
When analyzing, I think there is only german that has the word center in it:
![image](https://github.com/mytourbook/mytourbook/assets/25235665/2ea48fdc-5abd-4b6c-8561-88db9bea7ef5)

If it is meant to have the word center, the variable name `Map_Bookmark_Column_Longitude_Tooltip` and `Map_Bookmark_Column_Latitude_Tooltip` is misleading  but I can revert it to `Mapcenter` (though it should have a space between `Map` and `center`) and update the french translation.

Let me know

Reply:
`Mapcenter` was also not in the initial implementation

![image](https://github.com/mytourbook/mytourbook/assets/1283445/0519e86e-dd7d-440f-8d28-dace8344332c)

But sometimes I changed this wording slightly to better understand this value. It's difficult to find, when I changed this wording without creating new text key's, so I'll do this now after I merged your PR